### DISCUSSION TITLE
挂载本地目录,所有office类文档，都无法正常预览

### DISCUSSION THREAD

**Discussion by hostemail** [AlistGo/alist#252] [Q&A] [answered]
挂载本地目录,所有office类文档，如ppt，doc等，都无法正常预览，试了2.0多个版本，均是如此。
![image](https://user-images.githubusercontent.com/45652715/147628997-4a47edc4-4f67-4642-ab88-2a5be3059e8e.png)

![image](https://user-images.githubusercontent.com/45652715/147628967-286b9d68-0931-4040-a548-3c9829323aae.png)

![image](https://user-images.githubusercontent.com/45652715/147629020-d168e789-3cdf-4e3e-a266-30019aa95ed2.png)

**Comment by xhofe** [comment]
请提供一下具体文件地址

**Reply by hostemail** [reply]
最新版，beta版，多个版本都试了，本地挂载的目录，文档都不能预览。挂载阿里网盘的文档可以正常预览。

**Comment by xhofe** [comment, marked as answer]
Office Web Viewer 的预览必须使得office服务器可以访问到你的文件，因此必须是公网环境，它可能并不支持ipv6。
参考： https://www.microsoft.com/en-us/microsoft-365/blog/2013/04/10/office-web-viewer-view-office-documents-in-a-browser/

**Comment by xhofe** [comment]
阿里的office预览使用的是阿里的sdk，并不需要公网环境。

**Reply by hostemail** [reply]
哦，原理如此，那看来本地目录的文档没法预览了。多谢解答。

**Reply by xhofe** [reply]
在公网服务器上的可以，本地机器是不行的了
